taskcluster/ci/nightly-verify/kind.yml

```diff
@@ -0,0 +1,51 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+---
+loader: taskgraph.loader.transform:loader
+
+transforms:
+    - taskgraph.transforms.job:transforms
+    - taskgraph.transforms.task:transforms
+
+jobs:
+        
+    test:
+        description: Run a simple UI test to verify Nightly build can start 
+        worker-type: b-android
+        attributes:
+            build-type: debug
+            code-review: true
+        run-on-tasks-for: [github-pull-request, github-push]
+        run:
+            using: run-commands
+            use-caches: false
+            commands:
+                - [wget, {artifact-reference: '<signing/public/build/x86/geckoNightly/target.apk>'}, '-O', app-x86.apk]
+                - [wget, {artifact-reference: '<signing/public/build/armeabi-v7a/geckoNightly/target.apk>'}, '-O', app-arm.apk]
+                - [wget, {artifact-reference: '<signing-android-test/public/build/noarch/geckoNightly/target.apk>'}, '-O', android-test.apk]
+                - [automation/taskcluster/androidTest/ui-test.sh, x86-start-test, app-x86.apk, android-test.apk, '-1']
```

Comment:
verifying both x86 and arm

Reply:
My kind file has 3 extra properties within the treeherder config:
```
    kind: test
    platform: 'nightly-start-test/opt'
    symbol: nightly(ui-test-x86)
    tier: 2
```
Do I drop those 3 extra params or push them up to the `job-defaults` section?

Reply:
Yes. Push all of them up, except symbol. The symbol must be unique, so you need it defined in each job